View/ChannelForm.cs

```diff
@@ -56,10 +77,6 @@ private void OnBrowserAddressChanged(object sender, AddressChangedEventArgs e)
                 }
                 else
                 {
-                    if (Channel.Token != null)
```

Comment:
I left this here for a case where changing settings on the racetime site would drop your creds, best to leave this

Reply:
I'm not sure why this would be needed in this context though? There is no `RequestHandler` anymore, request interception is done though an event on the current form which can always directly read `Channel.Token.AccessToken`

Reply:
Solid that should be fine then.